source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -283,6 +285,21 @@ def _filter_too_much_connections(journeys, instance, request):
                 mark_as_dead(j, "too_much_connections")
 
 
+def _filter_min_transfers(journeys, instance, request):
+    """
+    eliminates journeys with number of connections less then min_nb_transfers among journeys
+    """
+    logger = logging.getLogger(__name__)
+    min_nb_transfers = get_or_default(request, 'min_nb_transfers', 0)
```

Comment:
On that, a non-ambiguous way may be to have a `min_nb_pt_section` param, maybe deprecate `max_nb_transfers` to a straightly tied (+1) param `max_nb_pt_section`.
It's always that difference between what's easy to integrate and what we think should be shown to traveler.